server/supernova/sc/sc_synth.cpp

```diff
@@ -182,6 +182,25 @@ float sc_synth::get(slot_index_t slot_index) const
     return mControls[slot_index];
 }
 
+bool sc_synth::getMappedSymbol(slot_index_t slot_index, char * str) const
+{
+    // analogous place in scsynth: SC_Group.cpp: void Group_QueryTreeAndControls(Group* inGroup, big_scpacket *packet)
+    bool isMapped = mMapControls[slot_index] != mControls+slot_index;
+    if (isMapped) {
+        if (mControlRates[slot_index] == 2) {
+            int bus;
+            bus = mMapControls[slot_index] - mNode.mWorld->mAudioBus;
+            bus = bus / mNode.mWorld->mBufLength;
+            sprintf(str, "a%d", bus);
+        } else {
+            int bus;
+            bus = mMapControls[slot_index] - mNode.mWorld->mControlBus;
```

Comment:
this is fine, for future reference you can declare while initializing (int bus = ...), this will save space and reduce visual noise

Reply:
you probably know that already, just stating my preference :)